test/simulator/timestamp/src/session_simulator.cpp

```diff
@@ -174,6 +225,10 @@ session_simulator::decode_timestamp_config_map(std::map<std::string, std::string
     timestamp_manager *ts_manager = &timestamp_manager::get_timestamp_manager();
     auto pos = config_map.find("commit_timestamp");
     if (pos != config_map.end()) {
+        /* Check that the timestamp string has valid hexadecimal characters. */
```

Comment:
We are having this check at multiple places, can we make it a function?

Reply:
I've moved these into a function in the timestamp manager.